<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-repeat: no-repat;
background-color:rgb(0, 0, 0)</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="styleSheet">
        <string notr="true">background: url(:/Resources/fondo.png);
background-repeat: no-repat;
background-color:rgb(0, 0, 0)</string>
       </property>
       <widget class="QWidget" name="Menu" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>600</width>
          <height>400</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>400</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QLabel" name="lbl_titulo">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>50</y>
           <width>131</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>32</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background: &quot;&quot;;</string>
         </property>
         <property name="text">
          <string>Galcon</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_jugar">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>80</y>
           <width>131</width>
           <height>32</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color : rgb(36, 81, 143);
border-radius: 9px; 
border-width: 1px; 
border-style: inset; 
border-color: rgb(36, 81, 143);
background: qlineargradient(spread:pad, x1:0.32, y1:0, x2:0.335, y2:1, stop:0 rgba(101, 101, 101, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>Partida RÃ¡pida</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_jugarOnline">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>240</y>
           <width>131</width>
           <height>32</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color : rgb(36, 81, 143);
border-radius: 9px; 
border-width: 1px; 
border-style: inset; 
border-color:  rgb(36, 81, 143);
background: qlineargradient(spread:pad, x1:0.32, y1:0, x2:0.335, y2:1, stop:0 rgba(101, 101, 101, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>Partida Online</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_salir">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>290</y>
           <width>131</width>
           <height>32</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color : rgb(36, 81, 143);
border-radius: 9px; 
border-width: 1px; 
border-style: inset; 
border-color: rgb(36, 81, 143);
background: qlineargradient(spread:pad, x1:0.32, y1:0, x2:0.335, y2:1, stop:0 rgba(101, 101, 101, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>Salir</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="styleSheet">
        <string notr="true">background: url(:/Resources/fondo2.png);
background-repeat: no-repat;
background-color:rgb(0, 0, 0)</string>
       </property>
       <widget class="QPushButton" name="plan1Prueba">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>170</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background: none;
border: none;
color: rgb(20, 27, 33);
</string>
        </property>
        <property name="text">
         <string>100</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Resources/miplaneta.png</normaloff>:/Resources/miplaneta.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="plan2Prueba">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>20</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: none;
border: none;
color: rgb(20, 27, 33);
background-image:url(:/Resources/suplaneta.png);
</string>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>380</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: none;</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="porcentaje">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>380</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: none;
border: none;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="minave">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>200</y>
          <width>51</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:none;
background-image:url(:/Resources/minave.png);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
